bin/config.py

```diff
@@ -58,13 +58,41 @@
 
 # Below here is some global state that will be filled in main().
 
-# Some default values needed for tests.
 args = argparse.Namespace()
-args.testcases = None
-args.cpp_flags = None
-args.verbose = None
-args.force_build = None
-args.error = None
+
+
+def set_default_args():
+    # Set default argument values
+    for arg in [
```

Comment:
If you can get the autoformatter to pack this list instead of putting it on many lines I think that'd be nice.

Reply:
Done, see 87b84ca